esp/templates/program/modules/creditcardmodule_stripe/failure.html

```diff
@@ -11,19 +11,17 @@
 {% endblock %}
 
 {% block content %}
-<h1>Credit Card Payment for {{ program.niceName }}</h1>
+<div style="background-color:#D3D3D3;">
+<h1>Credit Card Payment for {{ program.niceName }} <font color="red">FAILED</font></h1>
 
 {% load render_qsd %}
 
 {% inline_program_qsd_block program "payment_failure" %}
 
-<h3>Failure</h3>
-
-<p>Your credit card transaction was not successful.  More information about this is shown below.
+<p>Your credit card transaction was not successful.
 </p>
-
 {% if error_type == "inconsistent_po" %}
-<b>Inconsistent payment information.</b> Please contact the administrators with the following information.
+<b>Inconsistent payment information.</b> Please <a href="mailto:tech@learningu.org">contact the administrators</a> with the following information.
```

Comment:
Is this the correct email?  I would have expected we should tell them to contact the chapter.

Reply:
@hwatheod? My experience is that most admins just pass it along when there is a credit card error

Reply:
The po is a delimited string consisting of the program id and the user id.  It is formed entirely from our site data and has nothing to do with Stripe.  If there is an inconsistency, it seems like it's got to be a bug on the site, and admins probably can't do anything about it.

However, do we want students/parents to contact tech@learningu.org directly? If a lot of users on one chapter are experiencing this bug (or any bug, really), it might be better to get one consolidated report from the chapter, rather than lots of reports from different students/parents.

Reply:
Also one more unrelated issue I just noticed, in the {% if %} below this one, it says `{% if error_type == "inconsistent_po" %}` again.  It should be "inconsistent_amount".

Reply:
Okay, thanks for that clarification. That makes sense, then, to leave it as the chapter address. Thanks for the catch as well -- I'll change the following `if` as well.